package.json

```diff
@@ -13,6 +13,7 @@
     "autoprefixer": "6.3.6",
     "browser-sync": "2.12.8",
     "cssnano": "3.5.2",
+    "fs-extra": "^0.30.0",
```

Comment:
Let's use here a strict version `"0.30.0" - sometimes it leads to weird situations.